recipes/scanpy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "scanpy" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+build:
+  number: 0
+  skip: True # [not py36]
+source:
+  url: https://github.com/theislab/scanpy/archive/{{ version }}.tar.gz
+  sha256: ae29eb6e0998cb8eb613f72800a97265362c95bbe3cd4019d4127e006eef6453
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - seaborn
```

Comment:
I guess most of these dependencies, besides setuptools and python, can be removed from the host section, isn't it?